src/domain/Defaults/Defaults/Defaults.tsx

```diff
@@ -1,9 +1,40 @@
-import React from 'react'
+import isNumber from 'lodash/isNumber'
+import isString from 'lodash/isString'
+import React, { useContext } from 'react'
 
 interface IDefaults {
   /** Anchor component used for clickable links */
   AnchorComponent?: React.ComponentType<any>
   tenorApiKey?: string
+  i18nTranslationFunction?: any
+}
+
+const i18nDefaults = {
+  'delete': 'Delete',
+  'noResults': 'No Results',
+  'notProvided': 'Not Provided',
+  'filter': 'Filter',
+  'searchPlaceholder': 'Search',
+  'selectPlaceholder': 'Please Select',
+  'requiredSuffix': 'required',
+  'addFilterDropdownMenu.selectAFilter': 'Select A Filter',
```

Comment:
is it meant to be Select A Filter or Select a filter ?

Reply:
I just copied what the current string was, we can update it later.